### ISSUE TITLE
It seems a dev dependency is leaking: Cannot find module '@firebase/testing'

### ISSUE THREAD

**Issue by kariem** [freshfox/firestore-storage#3]
When deploying the function to Firebase, I get the following error:

```
Function failed on loading user code. Error message: Provided module can't be loaded.
Did you list all required modules in the package.json dependencies?
Detailed stack trace: Error: Cannot find module '@firebase/testing'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/srv/functions/node_modules/firestore-storage/dist/lib/test/test.js:11:18)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
```

Using
- firebase-tools 7.0.2
- firebase-admin 8.2.0
- firebase-functions 3.0.2

**Comment by kariem** [comment]
I think moving `src/lib/test` to `src/test` and updating the imports should be enough.

**Comment by dominicbartl** [comment]
FirestoreRuleTest was exported from the main `index.ts` file so the package `@firebase/testing` was loaded when importing any class from this package. Since this is just for [running tests against the Firestore rules with the emulator](https://github.com/freshfox/firestore-storage#running-tests-on-your-firestorerules) I removed the class from being exported through index.ts. v2.6.2 should fix this now.
Since `@firebase/testing` is only required for running those tests, I removed at from the peerDependenvies. This will remove the warning when installing the package. Instead I added a better documentation to the README.md
